### ISSUE TITLE
Texture memory corruption with nvidia on windows

### ISSUE THREAD

**Issue by mark-sgs** [coronalabs/corona#154]
Using any of the display.capture functions causes big problems.  Saved textures are being replaced with imagesheets from the build.  Over a short amount of time display objects are replaced with black rects and newTextures() get very corrupted.

This always happens and the pattern always repeats the same imagesheets in the same order.

[The is best demonstrated with a short video](https://www.spheregamestudios.com/corruption.gif).  (file too big to post here)

**Comment by pouwelsjochem** [comment]
We have also seen this issue in our Windows builds with the following GPU's, so not just Nvidia:

Radeon RX560
Intel UHD Graphics 630
Nvidia GeForce GTX 750 Ti
Nvidia GeForce GTX 965m
Nvidia GeForce GTX 1060
Nvidia GeForce GTX 1650
Nvidia GeForce GTX 1650 with Max-Q Design
Nvidia GeForce GTX 1660 Ti
Nvidia GeForce GTX 2060
Nvidia GeForce RTX 2070 SUPER

**Comment by pouwelsjochem** [comment]
Hey @Shchvov. Since this is a rather big issue which quite a few of our Windows users are experiencing, and you are busy with other stuff, I'm willing to have a look at it myself. 

Do you have any idea what could cause this issue? It would definitely help me if we could narrow down the scope.

Since I can't test it myself I'd probably require assistance from @mark-sgs to test the builds.

**Comment by Shchvova** [comment]
Problem is I don’t know where to look for solutions. Or causes, for that matter :(

**Comment by mark-sgs** [comment]
This problem went away for me eventually so I am not sure if Solar changed, Windows updated or nVidia updated.

**Comment by pouwelsjochem** [comment]
Oh interesting @mark-sgs , would you be willing to quickly try out an older build(s) to exclude Solar2D from the possible changes?

**Comment by mark-sgs** [comment]
I'm using windows 10 version 2004 (os build 19041.572) and nvidia version 456.71

**Comment by pouwelsjochem** [comment]
I'm mostly interested in if you upgraded Solar2D in the time between having the problem and not having it anymore. 

If the problem comes back in an older version (or perhaps you still remember what exact version you were using while starting this issue?) It could mean I don't have to spend time to find the cause. It would be great if you could install that older version to verify if the problem comes back.

**Comment by Shchvova** [comment]
I don't think there was any relevant changes in Solar2D for a while now...

**Comment by mark-sgs** [comment]
> I'm mostly interested in if you upgraded Solar2D in the time between having the problem and not having it anymore.

Of course, I've updated with each new build.  I suspect a windows update fixed things.

Long story short, no issues with latest Corona.
